gems/pending/util/system/evm_watchdog.rb

```diff
@@ -35,6 +39,13 @@ def self.check_evm
     end
   end
 
+  def self.check_for_update
+    return unless File.exist?(UPDATE_FILE)
+    packages = File.read(UPDATE_FILE).split(",")
+    LinuxAdmin::Yum.update(*packages)
```

Comment:
FYI, I believe this call was done in a rails context before.  We're pure ruby here so any libraries we need won't be required or autoloaded.